docs/source/contributing.rst

```diff
@@ -282,6 +284,33 @@ See an example of how an index should look like:
     Note that in this examples we group ``audio_voice1`` and ``audio_voice2`` in a single clip because the annotation ``voice-f0`` annotation corresponds to their mixture. In contrast, the annotation ``voice-f0`` is extracted from the multiclip mix and it is stored in the ``multiclips`` group. The multiclip ``multiclip1`` has an additional clip ``multiclip1-mix.wav`` which may be the master clip, the final mix, the recording of ``multiclip1`` with another microphone.
 
 
+.. _upload_index:
+
+3. Uploading the index to an online repository
+----------------------------------------------
+
+To prevent an the exponential growth in size of the library, we are storing all indexes in an online repository, while removing these from the repository itself.
+For that task, we use Zenodo.
+To use a dataloader, users may retrieve the index running the ``dataset.download()`` function that is also used to download the dataset.
+To download only the index, you may run ``.download(["index"])``. The index will be automatically downloaded and stored in the expected folder in Soundata.
```

Comment:
![Screenshot 2024-05-23 at 10 25 15 AM](https://github.com/soundata/soundata/assets/97785477/c16dbb5c-b985-4307-a4d9-f59e866b1cb8)
In this we see that even though it downloads the index in the default folder, it says it's downloading the index in the initialized folder

Reply:
I think that now the message should be improved. Can you please check @tanmayy24?

Reply:
[](url)
<img width="1056" alt="Screenshot 2024-05-24 at 9 25 14 AM" src="https://github.com/soundata/soundata/assets/97785477/0f392a69-0a3f-40ef-9484-9e7f40fd6fde">
Yes, it works fine but when we are doing download for index it shouldn't say downloading rest files at /testing/

Reply:
indeed. Fixed now :) Maybe it's a little bit overkill to use `if` `elif` `else` but that's the easiest way I found to log more meaningful messages here.

Reply:
Thank you! Looks good to me :)))